workflow/automation/org/nesi/calc_rrups_single.sl

```diff
@@ -25,6 +25,8 @@ timestamp=`date +%Y%m%d_%H%M%S`
 REL=$1
 MGMT_DB_LOC=$2
 
+echo $@
```

Comment:
@jpa198 What are your thoughts about leaving the `echo`s in? I think it's easy enough to add them in yourself if you need that. I believe the commands are printed out in the logs.

Reply:
I'm generally against this change, as the majority of running these scripts will be done via the automated workflow. 
If you search the logs for the command used, that's enough to run them manually. 
In most cases if you're running them manually it's because there's an error you're debugging, and at that point you're usually committed enough to dig through the log for the command.

Also this is beyond the scope of the PR.

Reply:
ok, reverting it.